src/views/routes/users.js

```diff
@@ -61,16 +66,66 @@ router.post('/login', async (req, res) => {
   }
 });
 
+router.get('/:username', async (req, res) => {
+  try {
+    const { username } = req.params;
+    await getUser({ username });
+
+    responseHandler.handleSuccess(res, success.CODE);
+  } catch (error) {
+    const { message } = error;
+    errLogger.error(error);
+
+    if (message.includes(AUTH__EMAIL_DOES_NOT_EXIST)) {
+      responseHandler.handleError(res, notFound.CODE);
+    } else {
+      responseHandler.handleError(res, serverError.CODE);
+    }
+  }
+});
+
+router.post('/forgot-password', async (req, res) => {
+  try {
+    const { username } = req.body;
+
+    await sendRecoveryEmail(username);
+    responseHandler.handleSuccess(res, success.CODE);
+  } catch (error) {
+    const { message } = error;
+    errLogger.error(error);
+
+    if (message.includes(AUTH__EMAIL_DOES_NOT_EXIST)) {
```

Comment:
I would not expect a response to `POST /users/forgot-password` to yield an error message called `AUTH__EMAIL_DOES_NOT_EXIST`. Instead, I would expect something that indicates `AUTH__USER_DOES_NOT_EXIST`.

Reply:
Not sure what I was thinking when I created this error message *face palm*. Does not even make sense once I have re-read it. Thanks for the suggestion!